arguments-optional.js

```diff
@@ -0,0 +1,22 @@
+/**
```

Comment:
I need the URL of the code challenge so I don't have to go searching for it.

Reply:
always think about how to make it easier for your reviewer to look at your code.  More context is always better.

Reply:
https://learn.freecodecamp.org/javascript-algorithms-and-data-structures/intermediate-algorithm-scripting/arguments-optional 

Will include that from the get-go from now on.

Reply:
Can you updated the code?